Formula/pynvim.rb

```diff
@@ -0,0 +1,24 @@
+class Pynvim < Formula
+  include Language::Python::Virtualenv
+
+  desc "Python client and plugin host for Nvim"
+  homepage "http://pynvim.readthedocs.io/en/latest/"
+  url "https://files.pythonhosted.org/packages/7a/01/2d0898ba6cefbe2736283ee3155cba1c602de641ca5667ac55a0e4857276/pynvim-0.4.3.tar.gz"
+  sha256 "3a795378bde5e8092fbeb3a1a99be9c613d2685542f1db0e5c6fd467eed56dff"
+  license "Apache-2.0"
+
+  depends_on "python@3.9"
+
+  resource "wcwidth" do
+    url "https://files.pythonhosted.org/packages/59/04/87fc6708659c2ed3b0b6d4954f270b6e931def707b227c4554f99bd5401e/msgpack-1.0.2.tar.gz"
+    sha256 "fae04496f5bc150eefad4e9571d1a76c55d021325dcd484ce45065ebbdd00984"
+  end
```

Comment:
maybe these two?
```suggestion
  resource "greenlet" do
    url "https://files.pythonhosted.org/packages/72/7e/d8586068d47adba73afc085249712bd266cd7ffbf27d8bc260c33e9d6133/greenlet-1.1.1.tar.gz"
    sha256 "c0f22774cd8294078bdf7392ac73cf00bfa1e5e0ed644bd064fdabc5f2a2f481"
  end

  resource "msgpack" do
    url "https://files.pythonhosted.org/packages/59/04/87fc6708659c2ed3b0b6d4954f270b6e931def707b227c4554f99bd5401e/msgpack-1.0.2.tar.gz"
    sha256 "fae04496f5bc150eefad4e9571d1a76c55d021325dcd484ce45065ebbdd00984"
  end
```

Reply:
Concerning the resources, you are listing the resources, which are needed for the `setup.py` like in case of [black-setup.py](https://github.com/psf/black/blob/main/setup.py) vs. [black-forumula](https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/black.rb)?